cmd/juju/status/status.go

```diff
@@ -196,7 +200,29 @@ func (c *statusCommand) Run(ctx *cmd.Context) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
-	return c.out.Write(ctx, formatted)
+	err = c.out.Write(ctx, formatted)
+	if err != nil {
+		return err
+	}
+
+	if status.IsEmpty() {
```

Comment:
Invert this to avoid unnecessary indentation
if !status.IsEmpty() {
return nil
}
...